cmd/whitesourceExecuteScan.go

```diff
@@ -90,89 +126,183 @@ func newWhitesourceUtils() *whitesourceUtilsBundle {
 	return &utils
 }
 
+// whitesourceScan stores information about scanned projects
+type whitesourceScan struct {
```

Comment:
What about extracting this and related functions to an own package `pkg/whitesource/scan`?

Reply:
I've started another branch "ws-package" and this portion is already done. I would like to address this in a follow up PR in order not to invalidate the reviews on this PR.